charts/feathr-online/conf/pipeline.conf

```diff
@@ -0,0 +1,57 @@
+geoip_demo(ip as string)
```

Comment:
current lookup.json and pipeline.conf are testing files, should not be packaged into helm chart templates.